help/help.md

```diff
@@ -0,0 +1,55 @@
+% systemtap (1) Container Image Pages
+% Tomas Tomecek
+% September 6th, 2017
+
+# NAME
+systemtap — programmable system-wide instrumentation system running in a container.
+
+# DESCRIPTION
+This container image provides functionality of packages `systemtap`, `systemtap-client` and `systemtap-testsuite`, which contains plenty of examples.
+
+# USAGE
+In order to use systemtap successfully, it requires debug information for running kernel, namely package `kernel-debuginfo`. There are two ways to do this when running systemtap in this container:
+
+1. Install `kernel-debuginfo` and `kernel-devel` inside container.
```

Comment:
kernel-devel is apparently installed already

Reply:
No package kernel-debuginfo available.

Reply:
I'm glad you brought this up, wasn't sure if I should provide steps how to do that. Am on it!